packages/create-plugin/templates/common/_package.json

```diff
@@ -20,8 +20,8 @@
   "devDependencies": {
     "@babel/core": "^7.21.4",{{#if useCypress}}
     "@grafana/e2e": "^11.0.7",{{/if}}
-    "@grafana/eslint-config": "^7.0.0",{{#if usePlaywright}}
-    "@grafana/plugin-e2e": "^1.8.3",{{/if}}
+    "@grafana/eslint-config": "^8.0.0",{{#if usePlaywright}}
```

Comment:
I would expect this to [break scaffolded plugins](https://github.com/grafana/eslint-config-grafana/pull/41) as I don't think all peer dependencies will be available.

Reply:
Good catch! Think renovate will update this PR and remove this dep once [this](https://github.com/grafana/plugin-tools/pull/1271) pr is merged.

Reply:
Asked Renovate to rebase the PR and now eslint-config remains on 7.0